docs/modules/deployment/pages/core/containers.adoc

```diff
@@ -0,0 +1,94 @@
+
+[[container-deployment]]
+= Containerized Deployment
+:description: Learn how to install OpenNMS Horizon/Meridian in a containerized environment like Kubernetes or Red Hat OpenShift.
+
+If your organization uses Kubernetes or Red Hat OpenShift, OpenNMS makes a Helm chart available to simplify {page-component-title} deployment.
+For information on deploying in a Docker environment, see the Docker tabs in https://docs.opennms.com/horizon/latest/deployment/core/getting-started.html[installation and configuration].
+
+Each deployment through Helm charts has a single Core server, Grafana, a custom Ingress that shares the RRD files and some configuration files, and multiple Sentinels for flow processing.
```

Comment:
I don't believe we need this line
`Each deployment through Helm charts has a single Core server, Grafana, a custom Ingress that shares the RRD files and some configuration files, and multiple Sentinels for flow processing.`